jobclient/python/docs/source/usage.rst

```diff
@@ -112,6 +143,5 @@ errors, you must wrap requests in a ``try`` block like so:
 JobClient Reference
 -------------------
 
-.. py:module:: cookclient
```

Comment:
This intentional?

Reply:
@hdrodz97

Reply:
Yup, that way the auto-links [here](https://github.com/twosigma/Cook/pull/1597/files/8cb4422f0ec5fcea59d86b8a20e3afaa6494ada5#diff-37ea7aa92a5b1e633271696fd7e88583R76-R77) work